platforms/core-runtime/base-services/src/test/groovy/org/gradle/util/PathTest.groovy

```diff
@@ -139,6 +139,18 @@ class PathTest extends Specification {
         path.append(Path.path('relative:subpath')) == Path.path(':path:relative:subpath')
     }
 
+    def "appending empty paths"() {
```

Comment:
💅  This test is right in the middle between the nicely symmetric tests. I'd put it after the next one, just to keep the continuity.

Also, other tests are called `appends`, so it'd be nice to keep things uniform

These are ultimate level nitpicks